### ISSUE TITLE
Report MuTect version instead of GATK version when --version is passed

### ISSUE THREAD

**Issue by lbeltrame** [broadinstitute/mutect#5]
I added MuTect support to @chapmanb bcbio-nextgen, and I'm doing checks on MuTect's version to ensure it is at least 1.1.5 (due to the infamous Java 7 issue), however MuTect reports the GATK version it was based on, rather than its actual version.

It would be nice to have --version report the actual MuTect version (also useful for debugging).

**Comment by lbergelson** [comment]
Good catch, we'll try to resolve this. Dealing with the --version flag is done by the GATK core before MuTect is started, which means it will probably involve changes to the GATK core to fix this.  

In the interim, MuTect outputs its version on any successful run. (Actually, any run passes the initial GATK argument checking and starts the MuTect code.)  Look for a line that looks something like this 

```
INFO  16:42:25,049 MuTect - VERSION INFO: MuTect:1.1.6-1-g10b1ba9 Gatk:2.7-2-g6bda569 
```

Also, any GATK version from GATK 2.6 onwards is running on Java 7, so checking the gatk version is probably good enough to know if it is compatible or not.
